src/core/propertyTypes.js

```diff
@@ -85,7 +85,7 @@ function selectorParse (value) {
 function selectorAllParse (value) {
   if (!value) { return null; }
   if (typeof value !== 'string') { return value; }
```

Comment:
Should `value` also be wrapped in `Array.from()`?  What is the use case?  In case a `NodeList` instance is passed in?